digital-deck-ui/src/store/index.js

```diff
@@ -158,8 +158,8 @@ export default createStore({
         },
 
         drawCards({ commit, state }) {
-            let oldPlayerCards = state.playerCards;
-            let oldCardsInDeck = state.numCardsInDeck;
+            // let oldPlayerCards = state.playerCards;
+            // let oldCardsInDeck = state.numCardsInDeck;
```

Comment:
Can you please remove these lines?

Reply:
These were supposed to be for the unit tests, but I can remove them if we don't need the tests anymore